include/drivers/emmc.h

```diff
@@ -177,6 +178,6 @@ size_t emmc_erase_blocks(int lba, size_t size);
 size_t emmc_rpmb_read_blocks(int lba, uintptr_t buf, size_t size);
 size_t emmc_rpmb_write_blocks(int lba, const uintptr_t buf, size_t size);
 size_t emmc_rpmb_erase_blocks(int lba, size_t size);
-void emmc_init(const emmc_ops_t *ops, int clk, int bus_width);
+void emmc_init(const emmc_ops_t *ops, int clk, int bus_width, int cmd23);
```

Comment:
Your interface would be more future-proof if you made the extra argument a "flags" argument instead, with bit 0 indicating whether cmd23 was supported or not. You would also need a macro (or enum) to define the flag bit value. Similarly, the global var `emmc_cmd23` in the c file would be better as `emmc_flags`.